index.js

```diff
@@ -2441,6 +2441,30 @@
         return n < 0 || negativeZero(n) ? Nothing() : slice(0, -n, xs);
       });
 
+  //# reverse :: [a] -> [a]
+  //.
+  //. Returns the elements of the given list in reverse order.
+  //.
+  //. ```javascript
+  //. > S.reverse([1, 2, 3])
+  //. [3, 2, 1]
+  //.
+  //. > S.reverse('abc')
+  //. 'cba'
+  //. ```
+  S.reverse =
+  def('reverse',
+      {},
+      [List(a), List(a)],
+      function reverse(xs) {
+        var result = [];
+        if (_type(xs) === 'String') return reverse(xs.split('')).join('');
+        for (var idx = xs.length - 1; idx >= 0; idx -= 1) {
+          result.push(xs[idx]);
+        }
```

Comment:
Ah, because i had used 4-space indentation before it would have exceeded the configured line length limit to have it on the same line hehe.. Now it just so fits ;)